examples/semdedup_example.py

```diff
@@ -72,7 +72,7 @@ def main(args):
 
 
 def attach_args():
-    parser = ArgumentHelper.parse_semdedup_args(add_input_args=True)
+    parser = ArgumentHelper.parse_semdedup_args()
```

Comment:
I removed it because all of the CLI scripts need at least one of
```
argumentHelper.add_arg_input_data_dir()
argumentHelper.add_arg_input_file_extension()
argumentHelper.add_arg_input_file_type()
argumentHelper.add_arg_input_text_field()
argumentHelper.add_arg_id_column()
argumentHelper.add_arg_id_column_type()
```
which are all input arguments.

Reply:
Makes sense, thanks